README.md

```diff
@@ -186,6 +187,10 @@ return static function (ContainerConfigurator $containerConfigurator): void {
     $parameters->set(Option::EXCLUDE_RECTORS, [
         SimplifyIfReturnBoolRector::class,
     ]);
+    
+    // Need to add a single Rector, not adding the whole rule set?  
+    $services = $containerConfigurator->services();
+    $services->set(RemoveUnusedClassConstantRector::class::class);
```

Comment:
This is mentioned on lines above:
https://github.com/rectorphp/rector/#running-rector

This 2nd part should containts all non-mentioned options, not to repeat them

Reply:
I'd leave both: they are not conflicting

Reply:
What would be the reason to have the 1st config then?

It might be confusing where is which option.
Merging them would bother reader lot of options that are not needed.